docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+services:
+  app:
+    restart: always
+    image: docker-rwd:tag
```

Comment:
I think `:tag` should be dropped so that `docker run --rm -ti docker-rwd` works.